blocks/property-search-bar/common-function.js

```diff
@@ -0,0 +1,254 @@
+  let output = '';
+  switch (filterName) {
+    case 'MinBedroomsTotal':
+    case 'MinBathroomsTotal':
+      output = 5;
+      break;
+    case 'LivingArea':
+      output = SQUARE_FEET;
+      break;
+    case 'YearBuilt':
+      output = YEAR_BUILT;
+      break;
+    case 'ApplicationType':
+      output = ['For Sale', 'For Rent', 'Pending', 'Sold'];
+      break;
+    case 'Sort':
+      output = SORT_BY;
+      break;
+    default:
+      break;
+  }
+  return output;
+}
+
+/**
+ *
+ * @param {string} filterName
+ * @param {string} defaultValue
+ * @returns {string}
+ */
+function buildSelectOptions(filterName, defaultValue) {
+  const conf = getConfig(filterName);
+  let output = `<option value="">${defaultValue}</option>`;
+  if (Array.isArray(conf)) {
+    conf.forEach((el) => {
+      output += `<option value="${el.value}">${el.label}</option>`;
+    });
+  } else {
+    const labelSuf = `+ ${defaultValue.split(' ')[1]}`;
+    for (let i = 1; i <= conf; i += 1) {
+      const label = `${i} ${labelSuf}`;
+      output += `<option value="${i}">${label}</option>`;
+    }
+  }
+  return output;
+}
+
+/**
+ * @param {string} filterName
+ * @param {string} defaultValue
+ * @returns {string}
+ */
+function buildListBoxOptions(filterName, defaultValue) {
+  const config = getConfig(filterName);
+  let output = `<li data-value="" class="tooltip-container highlighted">${defaultValue}</li>`;
+  if (Array.isArray(config)) {
+    config.forEach((conf) => {
+      output += `<li data-value="${conf.value}" class="tooltip-container">${conf.label}</li>`;
+    });
+  } else {
+    const labelSuf = `+ ${defaultValue.split(' ')[1]}`;
+    for (let i = 1; i <= config; i += 1) {
+      const label = `${i} ${labelSuf}`;
+      output += `<li  data-value="${i}" class="tooltip-container">${label}</li>`;
+    }
+  }
+
+  return output;
+}
+
+export function addOptions(filterName, defaultValue = '', mode = '', name = '') {
+  const selectedHtml = mode === 'multi'
+    ? `<div class="select-selected" role="button" aria-haspopup="listbox" name=${name}>${defaultValue}</div>
+            `
+    : '';
+
+  return `<section>
+        <div>
+            <select class="hide" aria-label="${defaultValue}">${buildSelectOptions(filterName, defaultValue, mode)}</select>
+            ${selectedHtml}
+            <ul class="select-item" role="listbox">${buildListBoxOptions(filterName, defaultValue, mode)}</ul>
+        </div>
+    </section>`;
+}
+
+export function formatInput(string) {
+  return string.replace(/[/\s]/g, '-').toLowerCase();
+}
+
+export function getPlaceholder(country) {
+  return country === 'US' ? 'Enter City, Address, Zip/Postal Code, Neighborhood, School or MLS#' : 'Enter City';
+}
+
+export function addRangeOption(filterName) {
+  const config = { ...TOP_LEVEL_FILTERS, ...EXTRA_FILTERS };
+  const { label } = config[filterName];
+  const filterLabel = label.charAt(0).toLocaleUpperCase() + label.slice(1).toLowerCase();
```

Comment:
If the label needs to be displayed in a certain way, why not just put that in the label value?